### ISSUE TITLE
Huge lag when opening small log file 

### ISSUE THREAD

**Issue by cottton** [JetBrains/ideolog#180]
_What steps will reproduce the issue?_
1. put log file (see attachment) into project (no matter which path)
2. open logfile (all fine so far)
3. rename file to ".log" instead of ".txt"

Background: the log is written with docker logs {container} \> "path/to/the/file.log"
and contains the logs of a mariadb upgrade.

Now you COULD say that this is not a propper log.
But the point is: a small file can lag the editor. And you dont know what is in the file until you open it =)

```
PhpStorm 2023.3.3
Build #PS-233.14015.96, built on January 24, 2024
Runtime version: 17.0.9+7-b1087.11 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 6.5.0-10022-tuxedo
GC: G1 Young Generation, G1 Old Generation
Memory: 8192M
Cores: 16
Registry:
  run.processes.with.pty=TRUE
Non-Bundled Plugins:
  de.netnexus.camelcaseplugin (3.0.12)
  com.intellij.ideolog (222.3.1.0)
  ru.adelf.idea.dotenv (2024.1)
  com.dmarcotte.handlebars (233.11799.172)
  com.mnw.tabmover (1.5.2)
Current Desktop: KDE
```
[mariadb_docker_log_after_upgrade.txt](https://github.com/JetBrains/ideolog/files/14270981/mariadb_docker_log_after_upgrade.txt)

**Comment by JLuc** [comment]
Same issue here : huge lag when opening small .log file (100 lines of apache access.log) with one simple filter and one simple pattern and one simple log format

**Comment by darealfive** [comment]
I experienced the same lagging problems with relatively small log-files ~100 rows or so with the latest PHPstorm update PhpStorm 2023.3.4
Build #PS-233.14475.35, built on February 13, 2024

**Comment by twelife** [comment]
the same issue!!!!
```text
PhpStorm 2023.3.4
Build #PS-233.14475.35, built on February 13, 2024
Runtime version: 17.0.10+1-b1087.17 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 6
Registry:
  run.processes.with.pty=TRUE
  ide.experimental.ui=true
Non-Bundled Plugins:
  com.codeium.intellij (1.6.34)
  mobi.hsz.idea.gitignore (4.5.2)
  de.espend.idea.php.annotation (9.4.0)
  ru.adelf.idea.dotenv (2024.1)
```

**Comment by kencho51** [comment]
The same issue here:
```
PhpStorm 2023.1.4
Build #PS-231.9225.18, built on July 13, 2023
Runtime version: 17.0.7+10-b829.16 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 13.4.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 12
Metal Rendering is ON
Registry:
    debugger.new.tool.window.layout=true
    run.processes.with.pty=TRUE
    ide.experimental.ui=true

Non-Bundled Plugins:
    org.rust.lang (0.4.200.5421-231)
    org.toml.lang (231.8109.91)
    org.intellij.plugins.hcl (231.8109.91)
    com.intellij.ideolog (222.3.1.0)
    com.intellij.properties (231.8770.3)
    com.wakatime.intellij.plugin (14.3.10)
    ru.adelf.idea.dotenv (2023.1)
    zielu.gittoolbox (500.1.8+222)

```

**Comment by smirok** [comment]
Fixed in 222.3.2.0, duplicates #177
